.circleci/config.yml

```diff
@@ -0,0 +1,50 @@
+version: 2.1
+
+
+jobs: 
+   build-and-push-docker-image:
+    environment:
+      - IMAGE: taraxa/explorer
+    machine:
+      image: ubuntu-2004:202010-01
+      docker_layer_caching: false
```

Comment:
Why did you turn off docker layer caching?

Reply:
it is to avoid accumulating images, if you dont turn off DLC we'll need to implement some way to cleanup